NitroxPatcher/Patches/Dynamic/PlayerCinematicController_OnPlayerCinematicModeEnd_Patch.cs

```diff
@@ -1,16 +1,14 @@
-﻿using System.Reflection;
 using NitroxClient.Communication.Abstract;
 using NitroxClient.GameLogic.PlayerLogic;
 using NitroxClient.MonoBehaviours;
 using NitroxClient.MonoBehaviours.Overrides;
 using NitroxClient.Unity.Helper;
-using NitroxModel.Helper;
 
 namespace NitroxPatcher.Patches.Dynamic;
 
 public sealed partial class PlayerCinematicController_OnPlayerCinematicModeEnd_Patch : NitroxPatch, IDynamicPatch
 {
-    private static readonly MethodInfo targetMethod = Reflect.Method((PlayerCinematicController t) => t.OnPlayerCinematicModeEnd());
+    // private static readonly MethodInfo targetMethod = Reflect.Method((PlayerCinematicController t) => t.OnPlayerCinematicModeEnd());
```

Comment:
I disabled all patches related to this because of all the cinematic issues we found out (stuck in cinematics consistently) but will re-enable before merging